packages/foreman/rubygem-ruby-libvirt/rubygem-ruby-libvirt.spec

```diff
@@ -1,85 +1,60 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name ruby-libvirt
-%global gem_require_name libvirt
+%global gem_require_name %{gem_name}
 
+Name: rubygem-%{gem_name}
+Version: 0.8.0
+Release: 1%{?dist}
 Summary: Ruby bindings for LIBVIRT
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.7.1
-Release: 2%{?dist}
-Group: Development/Languages
-License: LGPLv2+
+License: LGPL-2.1-or-later
```

Comment:
This looks like the SPDX version of `LGPLv2+`. Likely missing from the license mapping.